scipy/stats/_continuous_distns.py

```diff
@@ -7330,47 +7330,46 @@ def _stats(self, a):
     def _entropy(self, a):
         return 1 - 1.0/a - np.log(a)
 
-    def _support_mask(self, x, a):
-        if np.any(a < 1):
-            return (x != 0) & super(powerlaw_gen, self)._support_mask(x, a)
-        else:
-            return super(powerlaw_gen, self)._support_mask(x, a)
```

Comment:
Actually, this shouldn't be totally removed. This was not quite right as-is. Really, it should check `x != 0` only for those `a < 1`.